specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/registries.json

```diff
@@ -534,6 +585,7 @@
     },
     "RegistryTrackedResource": {
       "required": [
+        "location",
```

Comment:
Don't think we need to add location here if the base class TrackedResource already has it as a required property.

Reply:
Yes this needs to be removed. This is a breaking change that causes RegistryTrackedResource objects to be un-creatable, because the required location field is never passed down to TrackedResource if it's a named field of a child object.